packages/react-composites/package.json

```diff
@@ -104,9 +111,14 @@
     "styled-components": "~5.2.1",
     "ts-jest": "^26.4.4",
     "ts-node": "^9.1.1",
+    "ts-loader": "^8.0.12",
     "typescript": "4.1.5",
     "uuid": "^8.1.0",
-    "webpack": "5.38.1"
+    "webpack": "5.38.1",
+    "webpack-cli": "4.7.2",
+    "webpack-dev-server": "3.11.2",
+    "dotenv-webpack": "~7.0.3",
```

Comment:
nit: alphabetize

Reply:
Oh I didn't add it manually, it was done by npm/rush automatically.